dev/tests/integration/testsuite/Magento/LoginAsCustomer/_files/login_as_customer_config_enable.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
```

Comment:
Have you tried to use a config fixture instead of data fixture? Please, take a look at the example below

https://github.com/magento/magento2/blob/256ff9b4c13ca767dd43e4a45a689d72d2eacab0/dev/tests/api-functional/testsuite/Magento/GraphQl/Catalog/ProductCanonicalUrlTest.php#L50

Reply:
@rogyar Yes, Tried that but it didn't change the config value. Disabled the config in admin side and tested the test case which enables the config value using `@magentoConfigFixture`. This always fails, saying the config is not enabled.